vic/vic_run/src/lakes.eb.c

```diff
@@ -26,6 +26,7 @@
 
 #include <vic_def.h>
 #include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
So, you don't consider initialize_lake() to be driver code?  That makes
sense to me (not considering it to be driver code).  However, the other
initialize_* functions are still sitting in the drivers/shared/ section.

I guess for now we can consider initialize_lake() to be "promoted" to
vic_run/ due to its being used by the lake module.

On Wed, Feb 10, 2016 at 3:50 PM, Joe Hamman <notifications@github.com>
wrote:

> In vic/vic_run/src/lakes.eb.c
> <https://github.com/UW-Hydro/VIC/pull/388#discussion_r52546478>:
>
> > @@ -26,6 +26,7 @@
> >
> >  #include <vic_def.h>
> >  #include <vic_run.h>
> > +#include <vic_driver_shared.h>
>
> that's fine too. As long as we don't have any driver code sitting in
> vic_run.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Hydro/VIC/pull/388/files#r52546478>.
>